src/http/validations/verifyDevice.js

```diff
@@ -0,0 +1,19 @@
+import validator from 'validator';
+import FieldErrors from './fieldErrors';
+
+const verifyDevice = (req, res, next) => {
+  const { deviceId } = req.body;
+
+  const fieldErrors = new FieldErrors();
+
+  if (deviceId == undefined || !deviceId || typeof (deviceId) !== 'string') fieldErrors.addError('deviceId', 'deviceId is a required string');
```

Comment:
Duplicate checks here, `!deviceId` is enough

Reply:
fixed